web/src/components/tasks/TaxInput.tsx

```diff
@@ -85,6 +85,9 @@ export const TaxInput = (props: Props): ReactElement => {
       .update()
       .then(() => {
         setIsLoading(false);
+        updateQueue
+          .queueTaxFilingData({ ...taxFilingData, state: "SUCCESS", registeredISO: undefined, filings: [] })
+          .update();
```

Comment:
Why is the tax filing data here being explicitly set to `SUCCESS`? We are not hitting the API to confirm that the tax filing state is successful right?

Reply:
Should we change it to Pending?  Or should we change the logic for it?

Reply:
Changed!